src/linagora.esn.controlcenter/app/app.module.js

```diff
@@ -12,7 +25,6 @@
     'esn.user-configuration',
     'esn.business-hours',
     'esn.configuration',
-    'esn.user-configuration',
     'esn.i18n',
     'esn.home-page'
```

Comment:
you need to add the dependency to the application menu (`esn.application-menu`)

Reply:
fixed!